data/states/AK.yml

```diff
@@ -0,0 +1,136 @@
+code: AK
+name: Alaska
+grade: -1
+categories:
+  - title: Economic Abuse Defined in State Laws
+    grade: 0
+    innovative_policy_idea: ~
+    honorable_mention: ~
+    resource_links:
+      - text: AS 47.17.035
+        url: http://www.akleg.gov/basis/statutes.asp#47.17.035
+    criteria_met: []
+  - title: Paid and Protective Leave
+    grade: 0
+    innovative_policy_idea: ~
+    honorable_mention:
+      description: Victims of crimes are granted protections from employer penalization if the victim is subpoenaed or requested by the prosecuting attorney to attend a court proceeding for the purpose of giving testimony, needs to report the crime to law enforcement or participates in an investigation of the offense.
+      text: AS 12.61.017
+      url: http://www.akleg.gov/basis/statutes.asp#12.61.017
+    resource_links:
+      - text: AS 12.61.017
+        url: http://www.akleg.gov/basis/statutes.asp#12.61.017
+    criteria_met: []
+  - title: Safe Workplaces
+    grade: 0
+    innovative_policy_idea: ~
+    honorable_mention: ~
+    resource_links: []
+    criteria_met: []
+  - title: Unemployment Insurance (UI) Accessibility
+    grade: 2
+    innovative_policy_idea: ~
+    honorable_mention: ~
+    resource_links:
+      - text: AS 23.20.379
+        url: http://www.akleg.gov/basis/statutes.asp#23.20.379
+      - text: 8 AAC 85.095(c)(6)
+        url: http://www.akleg.gov/basis/aac.asp#8.85.095
+      - text: AS Dept of Labor Unemployment Handbook
+        url: https://labor.alaska.gov/unemployment/documents/uihandbook.pdf
+      - text: "AS Benefit Policy Manual: Voluntary Leaving (2010) (VL 155.05-1)"
+        url: https://labor.alaska.gov/unemployment/bpm/Voluntary_Leaving.pdf
+    criteria_met:
+      - 20
+      - 22
```

Comment:
@sarahgreen These numbers correspond to the criteria ids from the `categories.yml` file.